src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ManagedIndexMetaData.kt

```diff
@@ -147,31 +65,47 @@ data class ManagedIndexMetaData(
             STEP_START_TIME to stepStartTime?.toString(),
             STEP_COMPLETED to stepCompleted?.toString(),
             FAILED to failed?.toString(),
+            POLICY_COMPLETED to policyCompleted?.toString(),
+            ROLLED_OVER to rolledOver?.toString(),
             INFO to info?.let { Strings.toString(XContentFactory.jsonBuilder().map(it)) },
             CONSUMED_RETRIES to consumedRetries?.toString()
         )
     }
 
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
+        // Order matters here as we are only trying to show what is needed for the customer
         builder
             .field(INDEX, index)
             .field(INDEX_UUID, indexUuid)
             .field(POLICY_NAME, policyName)
 
         if (policySeqNo != null) builder.field(POLICY_SEQ_NO, policySeqNo)
         if (policyPrimaryTerm != null) builder.field(POLICY_PRIMARY_TERM, policyPrimaryTerm)
-        if (state != null) builder.field(STATE, state)
-        if (stateStartTime != null) builder.field(STATE_START_TIME, stateStartTime)
-        if (transitionTo != null) builder.field(TRANSITION_TO, transitionTo)
-        if (actionIndex != null) builder.field(ACTION_INDEX, actionIndex)
-        if (action != null) builder.field(ACTION, action)
-        if (actionStartTime != null) builder.field(ACTION_START_TIME, actionStartTime)
-        if (step != null) builder.field(STEP, step)
-        if (stepStartTime != null) builder.field(STEP_START_TIME, stepStartTime)
-        if (stepCompleted != null) builder.field(STEP_COMPLETED, stepCompleted)
+        if (policyCompleted == true) {
+            builder.field(POLICY_COMPLETED, policyCompleted)
+            return builder
+        }
+
+        if (rolledOver == true) {
```

Comment:
Is there a reason we don't show if `rolledOver` is `false`? shouldn't this be really checking for `step` is rollover and if it then we should always be interested in this param.

Reply:
Yes, good catch. I think we can go with: If rolledOver is true always show, otherwise only show rolledOver value if we're in the Rollover action. Otherwise we probably don't care about rolledOver yet as we either a) haven't gotten to Rollover action or b) don't even have Rollover in our Policy.